lib/testUtils/createRuleTester.js

```diff
@@ -305,3 +306,8 @@ function processGroup(rule, schema, equalityCheck) {
 function spaceJoin() {
   return _.compact(Array.from(arguments)).join(" ");
 }
+
+module.exports = deprecate(
+  createRuleTester,
+  "createRuleTester deprecated, please use https://github.com/stylelint/jest-preset-stylelint instead"
```

Comment:
```suggestion
  "createRuleTester deprecated. See https://github.com/stylelint/stylelint/issues/4267"
```

`jest-preset-stylelint` is not ready to be used. So it's better to point to discussion instead.